docs/docker-entrypoint.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Copyright (C) 2016 Red Hat, Inc.
```

Comment:
2017 year here

Reply:
Sure :-). Does the docs build work though?